docs/gallery_tags.rst

```diff
@@ -0,0 +1,266 @@
+===========================================
+Distribution Properties
+===========================================
+
+.. _symmetric:
+
+Symmetric
+---------
+
+These distributions are symmetric about their center (or can be symmetric under parameter choices):
+
+- `Laplace <./distributions/gallery/laplace.html>`_
+- `Normal <./distributions/gallery/normal.html>`_
+- `Cauchy <./distributions/gallery/cauchy.html>`_
+- `Student's t <./distributions/gallery/students_t.html>`_
+- `Beta (when α = β) <./distributions/gallery/beta.html>`_
+- `Uniform <./distributions/gallery/uniform.html>`_
+- `Logistic <./distributions/gallery/logistic.html>`_
+- *(Also, note that circular distributions such as Von Mises are symmetric on the circle.)*
+
+For more information on symmetric distributions, see:  
+`Symmetric distribution  <https://en.wikipedia.org/wiki/Symmetric_distribution>`_.
```

Comment:
I think we can not link to, but we can mention that distributions have a `skewness` method.